packaging/docker/samples/python/app/server.py

```diff
@@ -17,19 +17,19 @@
 # limitations under the License.
 #
 
+import os
 from flask import Flask
 import fdb
-import os
 
 app = Flask(__name__)
 
 fdb.api_version(int(os.getenv('FDB_API_VERSION')))
-db=fdb.open()
+db = fdb.open()
 
-COUNTER_KEY=fdb.tuple.pack(('counter',))
+COUNTER_KEY = fdb.tuple.pack(('counter',))
 def _increment_counter(tr):
     counter_value = tr[COUNTER_KEY]
-    if counter_value == None:
+    if counter_value is None:
```

Comment:
I think `==` is the right way. Because `tr[key]` is actually a future (see [here](https://apple.github.io/foundationdb/api-python.html#reading-data)). The semantic of future `==` is different from `is`.

The same applies to line 42.

Reply:
See also https://github.com/apple/foundationdb/issues/330

Reply:
Thanks for the pointer, I revert this change back.